services/galley/src/Galley/API/Teams.hs

```diff
@@ -196,7 +197,14 @@ uncheckedDeleteTeam zusr zcon tid = do
         (ue, be) <- foldrM (pushConvDeleteEvents now membs) ([],[]) convs
         let e = newEvent TeamDelete tid now
         let r = list1 (userRecipient zusr) (membersToRecipients (Just zusr) membs)
-        pushSome ((newPush1 zusr (TeamEvent e) r & pushConn .~ zcon) : ue)
+        -- To avoid DoS on gundeck, send team deletion events in chunks
+        let chunks = List.chunksOf 128 (toList r)
```

Comment:
Make this configurable by flag.

Reply:
I'm not against making this configurable, but I don't think it's very worthwhile. What is the exact benefit you see? Or are you thinking in a logic of "one shall not put magic numbers anywhere into a code base" ?

Reply:
More of a logic of 'anything that can be useful to tune at runtime by SREs should be a flag'. We're already discussing about 50 vs 100ms, and I also think this should be adjustable based on performance and testing. Not having to recompile the code to be able to tune things is nice.